src/hydra/fetchJsonLd.js

```diff
@@ -16,6 +16,11 @@ export default function fetchJsonLd(url, options = {}) {
     options.headers.set('Accept', jsonLdMimeType);
   }
 
+  if (null === options.headers.get('Authorization')) {
```

Comment:
I don't think it's the good way to handle this. You should use a decorator as explained here IMO: https://github.com/api-platform/admin#add-authentication-support